services/crypto/secrets/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+title: Secret Management
+id: CCC.SecMgmt
+description: |
+  Secret Management is a tool provided by cloud service providers 
+  to securely stores, retrieves, and manages sensitive data such as
+  API keys, passwords, database credentials, encryption keys, and certificates
+  and makes them accessible only to authorized users or applications.
+release_details:
+  - version: "2024.09"
+    assurance_level: None
+    threat_model_url: None
+    threat_model_author: None
+    red_team: None
+    red_team_exercise_url: None
+    release_manager:
+      name: Damien Burks
+      github_id: damienjburks
+      company: Citi
+      summary: Initial release
+    change_log:
+      - "Test"
```

Comment:
Should we have "Test" here?

Reply:
All existing metadata.yml file have this "Test" I think after Damien did some changes.